packages/color-handle/README.md

```diff
@@ -1,45 +1,162 @@
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-handle?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-handle)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-handle?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-handle)
 
-```
+```bash
 yarn add @spectrum-web-components/color-handle
 ```
 
 Import the side effectful registration of `<sp-color-handle>` via:
 
-```
+```javascript
 import '@spectrum-web-components/color-handle/sp-color-handle.js';
 ```
 
 When looking to leverage the `ColorHandle` base class as a type and/or for extension purposes, do so via:
 
-```
+```javascript
 import { ColorHandle } from '@spectrum-web-components/color-handle';
 ```
 
-## Standard
+### Anatomy
+
+The color handle consists of several key parts:
+
+- A visual handle element that indicates the current position
+- An optional color loupe that appears above the handle when active
+- Touch-responsive interaction areas
+- Color display showing the current selected color
+- Opacity checkerboard pattern for transparent colors
+
+```html
+<sp-color-handle></sp-color-handle>
+```
+
+**Internal Structure**: The component renders an inner div with the background color and an `<sp-color-loupe>` element that appears when the `open` property is true and the component is not disabled.
+
+### Options
+
+#### Color
+
+The `color` property sets the visual color displayed within the handle. This accepts any valid CSS color format. The default color is `rgba(255, 0, 0, 0.5)` (semi-transparent red).
+
+```html
+<div style="display: flex; gap: 16px; align-items: center; margin: 16px 0;">
+    <!-- Hex color -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="#ff0000"></sp-color-handle>
+    </div>
+
+    <!-- RGB format -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="rgb(255, 0, 0)"></sp-color-handle>
+    </div>
+
+    <!-- RGBA format with transparency -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="rgba(255, 0, 0, 0.5)"></sp-color-handle>
+    </div>
+
+    <!-- HSL format -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="hsl(0, 100%, 50%)"></sp-color-handle>
+    </div>
+
+    <!-- Named colors -->
+    <div style="position: relative; height: 20px; margin: 20px;">
+        <sp-color-handle color="red"></sp-color-handle>
+    </div>
+</div>
+```
+
+**Transparency Support**: When using transparent colors, the handle displays an opacity checkerboard pattern background to clearly show the transparency level.
+
+### States
+
+#### Standard
```

Comment:
I dont think this is necessary...? @nikkimk are we showing default states in all components?

Reply:
I include the default if I used tabs for a visual comparison, and typically for variants more than states, but it's not necessarily a hill to die on for me.